assets/models/organelles/Flagellum.glb.import

```diff
@@ -24,6 +24,7 @@ meshes/ensure_tangents=true
 meshes/storage=0
 meshes/light_baking=0
 meshes/lightmap_texel_size=0.1
+skins/use_named_skins=true
```

Comment:
Have you tested this change extensively? This is related to: https://github.com/Revolutionary-Games/Thrive/issues/1259